### PR TITLE
Get buffer

### PR DESCRIPTION
**Pull request by brkabuye**
Updated docker-compose to look into local repo.
Changes are made on listener, implementing HTTP

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,10 +4,16 @@ services:
     environment:
     - DOCKER=1
     - DEBUG=0
-    image: gebeyond/sa-data-hose:adapter
+    build:
+         context: .
+         dockerfile: Dockerfile
```
Comment (sarastik2) [inline docker-compose.yml]: These changes should not be pushed to GitHub, since building the images locally is only for local testing. Once this branch is merged to master, the new docker images will be built automatically on Dockerhub and they can be pulled down using the configuration that already exists here.

File: docker-compose.yml [hunk 2]
```diff
@@ -4,10 +4,16 @@ services:
...
     ports:
     - "4444:4444/udp"
   listener:
-    image: gebeyond/sa-data-hose:listener
+    build:
+          context: .
+          dockerfile: Dockerfile.listener
     ports:
     - "5000:5000"
+    - "5001:5001"
```
Comment (sarastik2) [inline docker-compose.yml]: Why the change to 5001? The listener should still be running on 5000

Comment (denniscammer) [reply docker-compose.yml]: That should be removed. I added that when we were troubleshooting, but that should be removed now.
